Lottery.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall(){
+        Random random = new Random();
+        int numb = random.nextInt(100);
+        ColorSupplier ColorsSupplier = new ColorSupplier();
+        Colors color = ColorsSupplier.getRandomColor();
+        Ball Ball = new Ball(numb, color);

```

Comment:
The variable name `Ball` should follow camelCase naming conventions. Consider renaming it to `ball`.